.github/workflows/cfn-publish-resource.yaml

```diff
@@ -5,7 +5,7 @@ on:
     inputs:
       regions:
         description: "comma seperated AWS regions.Default is all regions."
-        default: "af-south-1,ap-east-1,ap-northeast-1,ap-northeast-2,ap-northeast-3,ap-south-1,ap-southeast-1,ap-southeast-2,ap-southeast-3,ca-central-1,eu-central-1,eu-north-1,eu-south-1,eu-west-1,eu-west-2,eu-west-3,me-south-1,sa-east-1,us-east-1,us-east-2,us-west-1,us-west-2,ap-south-2,ap-southeast-4,eu-central-2,eu-south-2,me-central-1"
+        default: "af-south-1,ap-east-1,ap-northeast-1,ap-northeast-2,ap-northeast-3,ap-south-1,ap-southeast-1,ap-southeast-2,ap-southeast-3,ca-central-1,eu-central-1,eu-north-1,eu-south-1,eu-west-1,eu-west-2,eu-west-3,me-south-1,sa-east-1,us-east-1,us-east-2,us-west-1,us-west-2,ap-south-2,ap-southeast-4,eu-central-2,eu-south-2,me-central-1,ca-west-1,il-central-1"
```

Comment:
I was hesitant in adding these as default values given that the majority of our resources will not be present in these regions until https://jira.mongodb.org/browse/CLOUDP-233717 is completed. WDYT?

Reply:
i think it's ok to so new publishes go there, and at some moment we can see which ones are missing and publish

Reply:
if we make this change, I guess it means that customers will be able to create a stack in `ca-west-1` and create e.g. a `stream instance` (the next one to be released) but not able to create a cluster. Is that correct? do we want such experience?

Reply:
>if we make this change, I guess it means that customers will be able to create a stack in ca-west-1 and create e.g. a stream instance (the next one to be released) but not able to create a cluster. Is that correct? do we want such experience?

regarding this, we already plan to publish new stream resources to the new regions (ca-west-1) anyway. If this experience seems like a concern, then that means we should hold off releasing these new resources to these regions altogether. IMO it should be fine to publish to new regions now.